resources/Initialize.sql

```diff
@@ -0,0 +1,40 @@
+/* 
+ * MySQL Script - initializeDataBase.sql.
+ * Create testDB database, student table and StudentDetails table.
+ */
+ 
+-- Create testDB database
+CREATE DATABASE IF NOT EXISTS testdb;     
+
+-- Switch to testDB database
+USE testdb;
+
+-- create student table and StudentDetails table in the database
+CREATE TABLE IF NOT EXISTS student (id INT, name VARCHAR(255), city VARCHAR(255), gender VARCHAR(255), PRIMARY KEY (id));
+
+
+
+-- Create testDB database
+CREATE DATABASE IF NOT EXISTS testdb1;     
```

Comment:
Shall we use meaningful names to testdb and testdb1

Reply:
Done